metadata/dashboard/dashboard_metadata.go

```diff
@@ -1755,6 +1929,11 @@ func (m *MetadataServer) Start(port string) {
 	router.POST("/data/taskruns", m.GetTaskRuns)
 	router.GET("/data/taskruns/taskrundetail/:taskRunId", m.GetTaskRunDetails)
 
+	router.POST("/data/triggers", m.GetTriggers)
```

Comment:
do we need add these endpoints to auth?

Reply:
nah these requests get intercepted before that, and then we authorize again for a specific resource.